package.json

```diff
@@ -20,11 +20,13 @@
   },
   "scripts": {
     "build": "ember build",
-    "start": "ember server",
+    "lint:js": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
+    "start": "ember serve",
     "test": "ember try:each",
     "deploy": "ember github-pages:commit --message \"Deploy gh-pages from commit $(git rev-parse HEAD)\"; git push; git checkout -"
   },
   "dependencies": {
+    "bower": "^1.8.2",
```

Comment:
WHY?

Reply:
yeah i was also confused by this ^^ as i said, this was just generated by ember-cli-update. `bower.json` only contains "ember-try-placeholder", do you know what this is about?

Reply:
Aah.. Bower is needed for testing older ember versions, which required ember as a bower package. But since this is only needed for testing, I don't understand why it is listed as dependency instead of devDependency..